commit 7d8f5232eba76666c623d07e4ae4401c00f43570 (refs/changes/61/5561/3)
Author: John Levon <john.levon@joyent.com>
Date:   2019-02-12T18:07:10+00:00 (8 months ago)
    
    OS-7576 ctfdump -c should include non-root types
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/usr/src/cmd/ctfdump/ctfdump.c b/usr/src/cmd/ctfdump/ctfdump.c
index e241c35b47..ec879fcc06 100644
--- a/usr/src/cmd/ctfdump/ctfdump.c
+++ b/usr/src/cmd/ctfdump/ctfdump.c
@@ -10,7 +10,7 @@
  */
 
 /*
- * Copyright (c) 2018, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  */
 
 /*
@@ -982,7 +982,7 @@ ctfdump_source(void)
 		    strerror(errno));
 	}
 
-	if (ctf_type_iter(g_fp, B_FALSE, ctfsrc_collect_types_cb,
+	if (ctf_type_iter(g_fp, B_TRUE, ctfsrc_collect_types_cb,
 	    idnames) == CTF_ERR) {
 		warnx("failed to collect types: %s",
 		    ctf_errmsg(ctf_errno(g_fp)));
